tests/scripts/thread-cert/message.py

```diff
@@ -156,6 +156,14 @@ def assertMleMessageIsType(self, command_type):
         assert(self.mle.command.type == command_type)
 
     def assertMleMessageContainsTlv(self, tlv_class_type):
+        """To confirm if Mle message contains the TLV type.
+
+        Args:
+            tlv_class_type: tlv's type.
+
+        Returns:
+            tlv: if contains the TLV, return it.
```

Comment:
I guess `tlv` should be the type of the returned value.

Reply:
fixed